backend/windmill-api/openapi.yaml

```diff
@@ -3750,6 +3750,48 @@ paths:
                   - job
                   - approvers
 
+  /w/{workspace}/jobs/input_history/h/{hash}:
+    get:
+      summary: list inputs for previous completed jobs
+      operationId: getInputHistoryByHash
+      tags:
+        - job
+      parameters:
+        - $ref: "#/components/parameters/WorkspaceId"
+        - $ref: "#/components/parameters/ScriptHash"
+        - $ref: "#/components/parameters/Page"
+        - $ref: "#/components/parameters/PerPage"
+      responses:
+        "200":
+          description: input history for completed jobs with this script hash
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/JobInput"
+
+  /w/{workspace}/jobs/input_history/p/{path}:
+    get:
+      summary: list inputs for previous completed jobs
+      operationId: getInputHistoryByPath
+      tags:
+        - job
+      parameters:
+        - $ref: "#/components/parameters/WorkspaceId"
+        - $ref: "#/components/parameters/ScriptPath"
+        - $ref: "#/components/parameters/Page"
+        - $ref: "#/components/parameters/PerPage"
+      responses:
+        "200":
+          description: input history for completed jobs with this script path
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/JobInput"
```

Comment:
you are missing one for flow. What we have used in the past is using /f/ instead for flows